lib/graphql/introspection/directive_type.rb

```diff
@@ -4,7 +4,5 @@
   field :name, !types.String, "The name of this directive"
   field :description, types.String, "The description for this type"
   field :args, field: GraphQL::Introspection::ArgumentsField
-  field :onOperation, !types.Boolean, "Does this directive apply to operations?", property: :on_operation?
```

Comment:
Why did you remove this and not deprecate it like [GraphQL JS](https://github.com/graphql/graphql-js/blob/0aa78f61a2dc150b5ea9ee4f50b68a736796f068/src/type/introspection.js#L92-L114)? This breaks ecosystem tools like [GraphiQL](https://github.com/graphql/graphiql/blob/master/src/utility/introspectionQueries.js#L29-L31) that have not yet updated.